airbyte-integrations/connectors/source-singlestore/src/main/resources/spec.json

```diff
@@ -0,0 +1,202 @@
+                "multiline" : true,
+                "order" : 3,
+                "always_show" : true
+              },
+              "client_key_password" : {
+                "type" : "string",
+                "title" : "Client key password",
+                "description" : "Password for keystorage. This field is optional. If you do not add it - the password will be generated automatically.",
+                "airbyte_secret" : true,
+                "order" : 4
+              }
+            }
+          },
+          {
+            "title" : "Verify Full",
+            "description" : "Use SSL/TLS for encryption, certificate verification, and hostname verification.",
+            "required" : [
+              "mode",
+              "ca_certificate"
+            ],
+            "properties" : {
+              "mode" : {
+                "type" : "string",
+                "const" : "verify-full",
+                "order" : 0
+              },
+              "ca_certificate" : {
+                "type" : "string",
+                "title" : "CA certificate",
+                "description" : "CA certificate",
+                "airbyte_secret" : true,
+                "multiline" : true,
+                "order" : 1
+              },
+              "client_certificate" : {
+                "type" : "string",
+                "title" : "Client certificate",
+                "description" : "Client certificate (this is not a required field, but if you want to use it, you will need to add the <b>Client key</b> as well)",
+                "airbyte_secret" : true,
+                "multiline" : true,
+                "order" : 2,
+                "always_show" : true
+              },
+              "client_key" : {
+                "type" : "string",
+                "title" : "Client key",
+                "description" : "Client key (this is not a required field, but if you want to use it, you will need to add the <b>Client certificate</b> as well)",
+                "airbyte_secret" : true,
+                "multiline" : true,
+                "order" : 3,
+                "always_show" : true
+              },
+              "client_key_password" : {
+                "type" : "string",
+                "title" : "Client key password",
+                "description" : "Password for keystorage. This field is optional. If you do not add it - the password will be generated automatically.",
+                "airbyte_secret" : true,
+                "order" : 4
+              }
+            }
+          }
+        ]
+      },
+      "replication_method" : {
+        "title" : "Replication method",
+        "description" : "Replication method to use for extracting data from the database. STANDARD replication requires no setup on the DB side but will not be able to represent deletions incrementally. CDC uses the Binlog to detect inserts, updates, and deletes. This needs to be configured on the source database itself.",
```

Comment:
I thought that Binlog is a MySQL-specific thing. Lets delete CDC from this message, if we don't support it (it is not presented in the enum)

Reply:
Removing, it copy from mysql, currently STANDARD is only supported, CDC will be a separate task